### ISSUE TITLE
Set up DB

### ISSUE THREAD

**Issue by kuoirene** [amosproj/amos2024ss05-knowledge-graph-extractor#15]
## User story
1. As a system
2. I need to have a database connected to the application
3. So that files and items can be stored in the backend locally.

## Acceptance criteria
* Initialize setup of database.

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
